RabbitMQ.xs

```diff
@@ -14,9 +14,9 @@
 #define __DEBUG_ENABLED__ 0
 
 #if __DEBUG_ENABLED__ == 0
-# define __DEBUG__(X) /* NOOP */
+ #define __DEBUG__(X) /* NOOP */
```

Comment:
I think that reversing this to `#if __DEBUG_ENABLED__` (true) as the first condition would be more readable.